File: GTTS/INL.TrainingService.Models/MatchingPerson_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class MatchingPerson_Item : IMatchingPerson_Item
    {
        public Int64? PersonID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public string NationalID { get; set; }
        public char Gender { get; set; }
        public string IsUSCitizen { get; set; }
        public DateTime DOB { get; set; }
        public int? POBCityID { get; set; }
        public string POBCityName { get; set; }
        public string POBStateName { get; set; }
        public string POBCountryName { get; set; }
        public string LastVettingTypeCode { get; set; }
        public DateTime? LastVettingStatusDate { get; set; }
        public string LastVettingStatusCode { get; set; }
        public string JobTitle { get; set; }
        public string RankName { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/matching-person_item.ts


import { IMatchingPerson_Item } from './imatching-person_item';

export class MatchingPerson_Item implements IMatchingPerson_Item {
  
	public PersonID?: number;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public NationalID: string = "";
	public Gender: string = "";
	public IsUSCitizen: string = "";
	public DOB: Date = new Date(0);
	public POBCityID?: number;
	public POBCityName: string = "";
	public POBStateName: string = "";
	public POBCountryName: string = "";
	public LastVettingTypeCode: string = "";
	public LastVettingStatusDate?: Date;
	public LastVettingStatusCode: string = "";
	public JobTitle: string = "";
	public RankName: string = "";
  
}






